composer.json

```diff
@@ -5,8 +5,9 @@
     "license": [
         "AGPL-3.0+"
     ],
+    "version": "2.1.4"
```

Comment:
```suggestion
```

This is not necessary, Composer/Packagist will automatically pull the version tag from the repository.